cmd/restic/cmd_prune.go

```diff
@@ -55,6 +58,7 @@ func init() {
 	cmdRoot.AddCommand(cmdPrune)
 	f := cmdPrune.Flags()
 	f.BoolVarP(&pruneOptions.DryRun, "dry-run", "n", false, "do not modify the repository, just print what would be done")
+	f.StringVarP(&pruneOptions.UnsafeNoSpaceRecovery, "unsafe-recover-no-free-space", "", "", "UNSAFE, READ THE DOCUMENTATION BEFORE USING! Try to recover a repository stuck with no free space. Do not use without trying out 'prune --max-repack-size 1' first.")
```

Comment:
Shouldn't this also say `trying out 'prune --max-repack-size 0' first`?

Reply:
Oh, indeed, I've missed that after fixing `--max-repack-size 0`.